scenarioo-server/src/main/java/org/scenarioo/configuration/SwaggerConfiguration.java

```diff
@@ -0,0 +1,24 @@
+package org.scenarioo.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
```

Comment:
this will enable it for everyone correct? Including all customers running scenarioo. Is this intended?

Reply:
I dont see a problem with enabling this for all customers. But if you think its a problem we can make it customizable through a profile easily. But by default I would like to have it enabled. I do not see any problem with it.

Reply:
Good point. We could make it configurable. Docket has an enable(boolean)-Method and we could use a value configured in application.properties to enable/disable Swagger.